runners/samza/src/main/java/org/apache/beam/runners/samza/SamzaRunner.java

```diff
@@ -61,7 +61,7 @@
 })
 public class SamzaRunner extends PipelineRunner<SamzaPipelineResult> {
   private static final Logger LOG = LoggerFactory.getLogger(SamzaRunner.class);
-  private static final String BEAM_DOT_GRAPH = "beamDotGraph";
```

Comment:
Please keep the dot rendering as it is. We rely on it in open source to view beam dags.